app/assets/javascripts/application.js

```diff
@@ -78,6 +78,7 @@
 //= require miq_explorer
 //= require qs
 //= require miq_timeline
+//= require remote_consoles/remote_console
```

Comment:
you don't need this if you don't include the `application`

Reply:
good catch, thanks!